src/Data/Algorand/Transaction.hs

```diff
@@ -118,15 +119,16 @@ data TransactionType
 
 -- | Constants for @OnComplete@.
 --
-type OnComplete = Word64
-
-onCompleteNoOp, onCompleteOptIn, onCompleteCloseOut, onCompleteClearState, onCompleteUpdateApplication, onCompleteDeleteApplication :: OnComplete
-onCompleteNoOp = 0
-onCompleteOptIn = 1
-onCompleteCloseOut = 2
-onCompleteClearState = 3
-onCompleteUpdateApplication = 4
-onCompleteDeleteApplication = 5
+type OnComplete = Text
+
+onCompleteNoOp, onCompleteCreate, onCompleteOptIn, onCompleteCloseOut, onCompleteClearState, onCompleteUpdateApplication, onCompleteDeleteApplication :: OnComplete
```

Comment:
Wuh.. I don't like this very long type signature...

Reply:
Yeah seems like leftover from long time ago. I've updated this to normal type signature, you can check this again.

Reply:
nice!